process-engine/process_engine_meta
# Feature/shutdown database connection

Depends on:
* https://github.com/essential-projects/data_model_contracts/pull/15
* https://github.com/essential-projects/datasource_adapter_base/pull/8
* https://github.com/essential-projects/datasource_adapter_postgres/pull/8
* https://github.com/essential-projects/bootstrapper_node/pull/5
* https://github.com/essential-projects/http_node/pull/5
* https://github.com/essential-projects/data_model/pull/14
* https://github.com/essential-projects/bootstrapper/pull/3
* https://github.com/essential-projects/http_extension/pull/3

Closes https://github.com/process-engine/process_engine_meta/issues/8

## What did you change?

I have removed the `--exit` flag in the test script commands in the `package.json`.

## How can others test the changes?

As long as the process terminates correctly when Jenkins runs the integration tests the changes made in this and depending feature branches should work as intended.

If the shutdown doesn't work correctly you might see a Jenkins job that runs noticeably longer than usual and doesn't progress beyond the actual `mocha` test execution.

[This run](https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fprocess_engine_meta/detail/feature%2Fshutdown_database_connection/4/pipeline/49) took 7:18 minutes which seems normal. The tests failing don't seem to fail because of the changes covered by the feature branches this test feature branch uses.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
